### PR TITLE
MM-49984: Displaying active search backend in system console

### PR DESCRIPTION
**Pull request by agnivade**
This will be shown in the system console to let users know
which search backend is active.

Implemented via adding an extra param in the ping response.

https://mattermost.atlassian.net/browse/MM-49984

```release-note
The database section in the system console now has an additional
read-only section which shows the active search backend in use.

This can be helpful to confirm which is the currently active
search engine when there are multiple of them configured.
```

search_backend

### CODE REVIEW

**Review by isacikgoz — DISMISSED** [review]

File: server/platform/services/searchengine/searchengine.go [hunk 1]
```diff
@@ -45,8 +45,19 @@ func (seb *Broker) GetActiveEngines() []SearchEngineInterface {
 	if seb.ElasticsearchEngine != nil && seb.ElasticsearchEngine.IsActive() {
```
Comment (harshilsharma63) [inline server/platform/services/searchengine/searchengine.go:45]: Can we add the `IsIndexingEnabled` check for Elasticsearch as well? If I enable ES search and disable DB search, and at MM server startup ES server is not running, MM isn't able to search anywhere but I see `elasticsearch` in the admin console.

**Review by harshilsharma63 — CHANGES_REQUESTED** [review]

File: server/platform/services/searchengine/searchengine.go (hunk 1 above)
Comment (agnivade) [reply server/platform/services/searchengine/searchengine.go:45]: Currently they mean the same thing:

```
func (es *ElasticsearchInterfaceImpl) IsActive() bool {
	return *es.Platform.Config().ElasticsearchSettings.EnableIndexing
}

func (es *ElasticsearchInterfaceImpl) IsIndexingEnabled() bool {
	return *es.Platform.Config().ElasticsearchSettings.EnableIndexing
}
```

But I think what you mean is to check that the client is initialized. Let me make that change.

**Comment by agnivade** [comment]
@harshilsharma63 PTAL

**Comment by agnivade** [comment]
Gentle reminder @harshilsharma63 :)

@isacikgoz - looks like somehow your review was dismissed by @calebroseland, so we need another one

**Comment by calebroseland** [comment]
> Gentle reminder @harshilsharma63 :)
> 
> @isacikgoz - looks like somehow your review was dismissed by @calebroseland, so we need another one

This is news to me.. 😬 (I don’t recall touching this PR on Friday)

**Comment by agnivade** [comment]
GH does strange things :P

**Review by isacikgoz — APPROVED** [review]
Oops

**Comment by harshilsharma63** [comment]
Checking this now.

**Review by harshilsharma63 — APPROVED** [review]
LGTM 👍

**Pull request merged**